### ISSUE TITLE
Typo in Tutorial "Data on Manifolds"

### ISSUE THREAD

**Issue by ahenkes1** [geomstats/geomstats#870]
The math mode for the following passage is not active:

[https://geomstats.github.io/01_data_on_manifolds.html](url)

> The non-linear spaces we consider are called manifolds. A manifold :math:`M` of dimension :math:`m` is a space that is allowed to be curved but that looks like an m-dimensional vector space in the neighborhood of every point.

**Comment by nguigs** [comment]
Hi, thanks for finding this!
It looks fine in the [notebook](https://github.com/geomstats/geomstats/blame/master/notebooks/01_data_on_manifolds.ipynb#L206) but doesn't convert to [html](https://github.com/geomstats/geomstats.github.io/blob/master/01_data_on_manifolds.html). Any idea why @ninamiolane ?

**Comment by ninamiolane** [comment]
Yes, @ahenkes1 thank you for spotting this!

I think that there is this discrepancy because we have not (yet) automatized the conversion from the notebooks to their corresponding html. The conversion ipynb -> rst is currently done manually via:
```
jupyter nbconvert --to rst notebooks/01_data_on_manifolds.ipynb
```
and then travis automatically does the conversion rst -> html, and builds the documentation website with sphinx:
https://github.com/geomstats/geomstats/blob/f4f8a124fdc14f855e68e3aeaf8ae93970606e7f/.travis.yml#L35

We should automatize the conversion ipnyb -> rst with the `jupyter nbconvert ` command with Travis too.

**Comment by ninamiolane** [comment]
Fixed by #959
